projects/js-packages/social-logos/README.md

```diff
@@ -0,0 +1,88 @@
+//...
+render() {
+    return <SocialLogo icon="twitter" size={ 48 } />;
+}
+```
+
+#### Props
+
+* `icon`: String - the icon name.
+* `size`: Number - (default: 24) set the size of the icon.
+* `onClick`: Function - (optional) if you need a click callback.
+
+## Notes & Pixel Grid
+
+The icon grid is based on [Gridicons](https://github.com/Automattic/gridicons) and adheres to the same rules. That is to say, the set is designed on a 24px base grid. That means logos will look their sharpest and crispest when SVGs are inserted with 24px width/height, or the icon font is used at `font-size: 24px;`. 
+
+Logos will also scale well to other sizes, like 18px (75% size), or 36px (150% size). Normally, using icon-sets outside of their pixelgrid is a surefire way to get fuzzy icons. This is also true in the case of this logo set, however unlike custom-designed icons, this is almost unavoidable in the case of logos. The problem is, every single logo is designed with its own dimensions. If we are to respect branding guidelines (which we should), no hinting or pixel-tuning is applied to any logo added to this set. Which means even at the base 24px size, logos could appear fuzzy and less than optimal. That is the way of the world, and a tradeoff between flexibility and respecting the original logo design on one hand, and pixel-perfect logos on the other hand. 
+
+So to summarize:
+
+- **Do** use Social Logos at 48px, 36px, 24px, 18px, 12px. Prioritize 24px or above if you can.
+- **Try to avoid** using Social logos at 16px, 17px, or any arbitrary pixel-size that's incompatible with the base 24px grid. For example, don't size the icon font in EMs. 
+
+## Building
+
+In the commandline, type `npm run build`. This will clean up, polish, and generate a build folder that contains the following:
+
+- A folder called `font`. This folder contains a WOFF2 font file as well as a CSS file with the font file included inline.
+- A folder called `php`. This folder contains a PHP file that allows one to retrieve a logo SVG using the `get_social_logo()` function.
+- A folder called `react`. This folder contains the minified SocialLogo component as well as an example component.
+- A folder called `svg-clean`. This folder contains minimized SVGs of every Social Logo. These SVGs can be used in Figma for mockups.
+- A folder called `svg-sprite`. This folder contains an SVG sprite called `social-logos.svg`, which can be referenced using `use`, as well as an example file.
+
+Do remember to update the React components where they are used when you add a new icon.
+
+## Publishing to NPM
+
+- Follow install instructions
+- Check in changes if any and follow PR process.
+- Bump package version in package.json to the next desired version and add an alpha postfix `1.1.0-alpha.1`
+- While testing changes publish using the next tag `npm publish --tag next`
+- If changes look good remove postfix in the version `1.1.0`
+- Publish using the latest tag `npm publish --tag latest`
```

Comment:
Nope. Just prepare a new version as documented at PCYsg-Np7-p2 and let the automation do the rest.